code/modules/mob/living/carbon/human/life.dm

```diff
@@ -32,6 +32,12 @@
 		return
 	//Update our name based on whether our face is obscured/disfigured
 	name = get_visible_name()
+	//makes sure admins are adminning instead of having fun grrr
+	if(ckey in list("Silicons", "CygnusB", "Putnam", "R3dtaile", "DeltaFire", "R0-adic", "Xantholne", "Trojancoyote", "Dzahlus", "NicoDeimos", "Bhijn", "BlueWildrose")) // Don't know if nico is still an admin, don't have an admin list anywhere so feel free to add if there's people missing
```

Comment:
"ckeys" are case sensitive, all should be lowercased or the automatic administrative detection system will fail.

Reply:
alternatively, you could check if the mob client has a "holder" which means the mob IS an administrator.

Reply:
> 
> 
> alternatively, you could check if the mob client has a "holder" which means the mob IS an administrator.

Too much effort. I've spent what little strength I have writing synonyms to "dick, ass, fuck, cum, tits, boobs"
I hope you can understand.

Reply:
It is done.